drivers/sensor/wsen_hids_2525020210002/Kconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 Würth Elektronik eiSos GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+
+config WSEN_HIDS_2525020210002
```

Comment:
Is `2525020210002` a part number or SKU? This is a long name for a driver

Reply:
This is a part number uniquely identifying the sensor as for example hids refers to the family of the sensor (humidity in this case), so it is important to have it part of the driver name.